src/Microsoft.AspNetCore.Mvc.Analyzers/SymbolNames.cs

```diff
@@ -58,5 +58,9 @@ internal static class SymbolNames
         public const string ProducesResponseTypeAttribute = "Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute";
 
         public const string RenderPartialMethod = "RenderPartial";
+
+        public const string RequiredAttribute = "RequiredAttribute";
```

Comment:
Fully qualify this and use that to look up the type.

Reply:
Will change